src/librustdoc/html/static/main.js

```diff
@@ -137,10 +138,6 @@ function defocusSearchBar() {
                 sidebar.appendChild(div);
             }
         }
-        var themePickers = document.getElementsByClassName("theme-picker");
```

Comment:
Why this removal?

Reply:
It doesn't actually do anything.

It's removing a `display` attribute from the `<div class=theme-picker>`, but that display attribute is never added to it. The display attribute gets added to `theme-choices`, making this code a no-op.

Reply:
I need to check it to confirm (tomorrow very likely). If it's true, I'm glad you found it! :D